vita3k/gui/src/controls_dialog.cpp

```diff
@@ -128,10 +149,17 @@ void draw_controls_dialog(GuiState &gui, EmuEnvState &emuenv) {
         set_controller_overlay_scale(emuenv.cfg.overlay_scale);
         config::serialize_config(emuenv.cfg, emuenv.cfg.config_path);
     }
+    ImGui::Spacing();
+    if (overlay_editing && ImGui::SliderInt("Overlay opacity", &emuenv.cfg.overlay_opacity, 0, 255, "%d")) {
+        set_controller_overlay_opacity(emuenv.cfg.overlay_opacity);
+        config::serialize_config(emuenv.cfg, emuenv.cfg.config_path);
+    }
```

Comment:
It would be better for the user to make the slider go from 0 to 1.0 and just multiply the value by 255 when storing it I think.

Reply:
Not really. I tried with 0 to 1.0, but it wasn't perfect because you can't control the slider well.
So basically you can't control opacity as you want with such small slider. 
With from 0 to 255 you can control opacity  poinwise.
Other emulators also use from 0 to 100, or from 0 to 255

Reply:
Ok, I think a 0 to 100 cursor should be fine then

Reply:
> Ok, I think a 0 to 100 cursor should be fine then

Done